### PR TITLE
Add platypus possibilities

### PR DESCRIPTION
**Pull request by chelauk**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by chelauk** [comment]
tested environment.yml with docker and it builds

**Comment by MaxUlysse** [comment]
Good news, what is this tool for?

File: environment.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ dependencies:
   - bioconda::htslib=1.9
   - bioconda::manta=1.6.0
   - bioconda::multiqc=1.8
+  - bioconda::platypus-variant=v0.8.1.2
```
Comment (MaxUlysse) [inline environment.yml]:
```suggestion
  - bioconda::platypus-variant=0.8.1.2
```
I don't think you need the `v`

Comment (MaxUlysse) [reply environment.yml]: cf https://bioconda.github.io/recipes/platypus-variant/README.html

**Comment by apeltzer** [comment]
https://www.nature.com/articles/ng.3036

Variant Caller, but had to look it up too :-)

**Comment by chelauk** [comment]
Platypus will combine the outputs from mutect to improve the variant calling

**Pull request closed without merging**